public/style/main.css

```diff
@@ -73,43 +78,47 @@ body{
     width: 28vw;
     height: 15vw;
     background-color: #ebe8e8;
-    border:dashed 4px #a0a0a0;
+    border: dashed 4px #a0a0a0;
     display: flex;
     justify-content: center;
     align-items: center;
     flex-direction: column;
     text-align: center;
 }
-.my-form>h4,#file-inp-label{
+
+.my-form>h4,
+#file-inp-label {
     text-align: center;
     color: #727171;
     font-size: 1.4vw;
     font-family: 'Nanum Gothic', sans-serif;
     font-weight: bold;
 }
-#file-inp-label{
+
+#file-inp-label {
     border-bottom: solid #727171 2px;
 
 }
 
 
-.my-form>label{
+.my-form>label {
     font-weight: 600;
 }
-#file-inp-label:hover{
+
+#file-inp-label:hover {
     cursor: pointer !important;
 }
 
-.my-form>h4{
+.my-form>h4 {
```

Comment:
I don't know where this **css** class comes from, but when I work on styles I always avoid using `!important`  unless extremely necessary, because being the ultimate "override", having too many of them can lead to unexpected results hahahahah

Reply:
Done!